src/components/FeedbackForms/MissingRecord/RecordPanel.tsx

```diff
@@ -0,0 +1,232 @@
+import { Field, FieldArray, FieldArrayRenderProps, FieldProps, useField, useFormikContext } from 'formik';
+import { useState } from 'react';
+import { KeywordList, ReferencesTable } from '.';
+import { AuthorsTable } from './AuthorsTable';
+import { Collection, IAuthor, FormValues } from './types';
+import { URLTable } from './URLTable';
+
+const collections: { value: Collection; label: string }[] = [
+  { value: Collection.astronomy, label: 'Astronomy and Astrophysics' },
+  { value: Collection.physics, label: 'Physics and Geophysics' },
+  { value: Collection.general, label: 'General' },
+];
+
+const datePropConfig = {
+  dateNavBtnProps: {
+    variant: 'outline',
+  },
+  dayOfMonthBtnProps: {
+    defaultBtnProps: {
+      borderColor: 'red.300',
+      _hover: {
+        background: 'blue.400',
+      },
+    },
+    selectedBtnProps: {
+      background: 'blue.200',
+      borderColor: 'blue.300',
+      color: 'blue.600',
+    },
+    todayBtnProps: {
+      variant: 'outline',
+    },
+  },
+};
+
+export const RecordPanel = ({
+  isNew,
+  formData,
+  onRecordLoaded = noop,
+}: {
+  isNew: boolean;
+  formData?: FormValues;
+  onRecordLoaded?: () => void;
+}) => {
+  const [recordLoaded, setRecordLoaded] = useState(formData ? true : false);
+
+  const {
+    values: { noAuthors, authors, urls, keywords, references },
+  } = useFormikContext<FormValues>();
+
+  const [] = useField<IAuthor[]>({
+    name: 'authors',
+    validate: (value: IAuthor[]) => {
+      if (!noAuthors && (!value || value.length === 0)) {
+        return 'Authors are required. Check "Abstract has no authors" box if no authors.';
+      }
+    },
+  });
+
+  const [, , { setValue: setPubDateValue }] = useField<Date>('pubDate');
+
+  return (
+    <Stack direction="column" gap={4} m={0}>
+      <Field name="bibcode">
+        {({ field }: FieldProps<FormValues['bibcode']>) => (
+          <FormControl isRequired>
+            <FormLabel>Bibcode</FormLabel>
+            <Flex direction="row">
+              <Input {...field} />
+              {!isNew && (
+                <Button
+                  size="md"
+                  borderStartRadius={0}
+                  borderEndRadius={2}
+                  isDisabled={!field.value || field.value.length === 0}
+                >
+                  Load
+                </Button>
+              )}
+            </Flex>
+          </FormControl>
+        )}
+      </Field>
+
+      {(isNew || (!isNew && recordLoaded)) && (
+        <>
+          <FormControl>
+            <FormLabel>Collection</FormLabel>
+            <CheckboxGroup>
+              <Field name="collection">
+                {({ field }: FieldProps) => (
+                  <Stack direction="row">
+                    {collections.map((c) => (
+                      <Checkbox key={`collection-${c.value}`} {...field} value={c.value}>
+                        {c.label}
+                      </Checkbox>
+                    ))}
```

Comment:
Could probably not bother with the config and loop pattern here, since it's unlikely to change frequently, we can just list out the three checkboxes to make it simpler.

Reply:
I used the list since for the expansion, I don't know what the collections look like. But for the next phase, I changed the type of the `database` field in the search response, instead of keeping a list here.